packages/utilities/moment-timezone-include/src/postInstall/index.js

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
```

Comment:
```suggestion
```

This file isn't executable (nor after compilation) and is run using `node {script.js}`, so you don't need a shebang
It doesn't technically hurt, but it is misleading